src/Analyzer.php

```diff
@@ -0,0 +1,24 @@
+<?php declare(strict_types=1);
+
+namespace Relewise;
+
+use Relewise\Infrastructure\HttpClient\Response;
+use Relewise\Models\ProductPerformanceRequest;
+use Relewise\Models\ProductPerformanceResponse;
+
+class Analyzer extends RelewiseClient
+{
+    public function __construct(private string $datasetId, private string $apiKey, private int $timeout = 120)
+    {
+        parent::__construct($datasetId, $apiKey, $timeout);
+    }
+    public function productPerformance(ProductPerformanceRequest $request) : ?ProductPerformanceResponse
```

Comment:
```suggestion

    public function productPerformance(ProductPerformanceRequest $request) : ?ProductPerformanceResponse
```

Reply:
Let's make that in a separate PR as it is auto-generated and would cause updates in all clients if changed.